randovania/games/fusion/logic_database/Sector 1 SRX.txt

```diff
@@ -427,6 +427,9 @@ Extra - room_id: [13]
   * Extra - door_idx: (29,)
   > Room Center
       Trivial
+  > Event - Stabilizer 3 Online
+      # Speed Boost the Stabilizer: https://youtu.be/V20W0F4dZP0
+      Level 0 Locks and Speed Booster and After Stabilizer 4 Restored and Shinespark Tricks (Intermediate) and Disabled Entrance Rando
```

Comment:
oh oops i forgot to mention, this also needs `Disabled door lock rando`

Reply:
Ah, I wasn't sure if this was needed. Added that in too.